packages/actor-init-query/lib/HttpServiceSparqlEndpoint.ts

```diff
@@ -550,13 +550,13 @@ export class HttpServiceSparqlEndpoint {
       return;
     }
 
-    const s = request.url;
+    const s = `http://localhost:${this.port}${request.url}`;
```

Comment:
Comunica will not always necessarily run on localhost.
There should be another property in `request` you can use. I don't remember which one though.